src/Microsoft.CodeAnalysis.FxCopAnalyzers/Setup/FxCopAnalyzersPackage.cs

```diff
@@ -11,8 +11,6 @@ namespace FxCopAnalyzersSetup
     class FxCopAnalyzersPackage : Package
 #pragma warning restore CA1812
     {
-#pragma warning disable CA1823 // False unused field diagnostic: https://github.com/dotnet/roslyn-analyzers/issues/1191
```

Comment:
I believe so, looks like a dupe of https://github.com/dotnet/roslyn-analyzers/issues/933.